extensions/dicom-microscopy/src/utils/Publisher.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Publisher allows applying pub-sub by extending it in other classes definition
+ */
+export default class Publisher {
```

Comment:
we have a pubsub service, please use that. Look at CornerstoneViewportService as an example.